api/docs/BUILD

```diff
@@ -36,6 +36,7 @@ proto_library(
         "//envoy/config/filter/http/ext_authz/v2:pkg",
         "//envoy/config/filter/http/fault/v2:pkg",
         "//envoy/config/filter/http/grpc_http1_reverse_bridge/v2alpha1:pkg",
+        "//envoy/config/filter/http/grpc_stats/v2alpha:pkg",
```

Comment:
@htuch is this not sufficient to wire into docs generation?

Reply:
I'm confused then why we don't see v3 protos; do you have them locally after a `fix_format`?

Reply:
@htuch does check_format look for missing files also in the diff?

Reply:
I don't think I saw extra files. I usually run check_format fix before sending a PR.

Reply:
I'll take an action item to figure out what went wrong here, as this will also affect other PRs. I have an item in my TODO already to try and get some better separation of the docs target and what we use for `protoxform`.